source/plugin/configuration/serialization.rst

```diff
@@ -135,6 +164,80 @@ The ``@ConfigSerializable`` annotation eliminates the need for any registration
 just generate an :javadoc:`ObjectMapper` for the class. The only limitation is that Configurate needs an empty
 constructor to instantiate a new object before filling in the annotated fields.
 
+.. note::
+
+    You can also have fields that are not are not annotated with ``@Setting`` in your ``@ConfigSerializable`` classes.
+    These fields won't be persisted to config files and can be used to store temporary references for your plugin.
+
+Using Default Values in ConfigSerializable Types
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is also possible to use default values inside of ``@ConfigSerializable`` types, you just have to use Java's field
```

Comment:
This is a rather convoluted sentence. I suggest to end the sentence at "types" and start a new one at "You just ..."
(or alternatively turn the comma into a semicolon, the semicolon into a full stop, and start a new sentence at "as long ...")